libraries/dawg-show/src/main/webapp/views/powerPrompt.jsp

```diff
@@ -29,6 +30,7 @@ Prompt for giving different options concerning power to an stb
         <font size="5px" color="#dedbdb"><b>What do you want to do?</b></font>
     </div>
     <div style="padding:30px;background-color:#dedbdb">
+        <input style='font-size:17px; display: <%= (Boolean) request.getAttribute("isXR11") ? "block;" : "none;" %>' type="button" value="Send Power Key" onclick="sendPowerKey()" />
```

Comment:
Don't think we want that case. We hid the power from partners because we didn't want them to be able to power off the device. If we have them also use the XR11 it would be enabled again. 
I think there might need to have a flag somewhere else for controlling whether or not the power button is displayed.
Maybe it is a different url path instead of simplified?